### PR TITLE
Feature/support bower package

### PR DESCRIPTION
**Pull request by paucls**
Added support for Bower. The project can be published in bower registry and any UI application using bower can install it directly.

### CODE REVIEW

File: src/angular-drag-drop.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
-var Angular = require('angular');
+var Angular;
 
-module.exports = 'filearts.dragDrop';
+if (typeof angular === 'object') {
```
Comment (ggoodman) [inline src/angular-drag-drop.js:3]: Can you comment on why these changes were necessary?

The build step `npm run build` will transform this file into a UMD-compatible module.

File: src/angular-drag-drop.js [hunk 2]
```diff
@@ -15,7 +23,7 @@ mod.factory('dragContext', ['$rootElement', function($rootElement) {
         return Angular.extend(context, {
             data: null,
             reset: reset,
-            start: start,
+            start: start
```
Comment (ggoodman) [inline src/angular-drag-drop.js]: I'd prefer that your PR not include stylistic changes.

File: dist/angular-drag-drop.js [hunk 3]
```diff
@@ -16,7 +24,7 @@ mod.factory('dragContext', ['$rootElement', function($rootElement) {
         return Angular.extend(context, {
             data: null,
             reset: reset,
-            start: start,
+            start: start
```
Comment (ggoodman) [inline dist/angular-drag-drop.js]: I'd prefer that your PR not include stylistic changes.

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  name: "ggoodman-angular-drag-drop",
```
Comment (ggoodman) [inline bower.json]: This file does not appear to be valid json (unquoted keys). Does bower support pseudo-json files?

**Review by ggoodman — CHANGES_REQUESTED** [review]
Thanks for getting the ball rolling. Please see the attached review feedback.

File: src/angular-drag-drop.js (hunk 1 above)
Comment (paucls) [reply src/angular-drag-drop.js:3]: 1) This hacky change here, with the if and else, is to support the use of the library from apps that don't use webpack (they expect to have 'angular' already available globally, not to be imported using require).
Maybe there is another way to accomplish this configuring direclty webpack output bundle. 

2) The change in the angular module name is to fix the name itself, reading the README it seems that the angular module is called "angular-drag-drop" instead of "filearts.dragDrop".
Again, if the app using the library does not support the use of require it needs to specify directly the angular module name to be imported like this:

`angular.module('my-app', ['angular-drag-drop']);`

instead of:

```
var Angular = require('angular');
var mod = Angular.module('yourModule', [
  require('angular-drag-drop'),
]);

```

File: src/angular-drag-drop.js (hunk 2 above)
Comment (paucls) [reply src/angular-drag-drop.js]: reverted

File: dist/angular-drag-drop.js (hunk 3 above)
Comment (paucls) [reply dist/angular-drag-drop.js]: reverted

File: bower.json (hunk 4 above)
Comment (paucls) [reply bower.json]: You are right. Fixed.

**Comment by ggoodman** [comment]
I'm not willing to compromise on the idiomatic commonjs format for the main entry point at this time. I would be willing to accept a PR whose scope is limited to `README.md` changes and a `bower.json` file.
